documentation/enforce/rules/lang/is_truthy/README.md

```diff
@@ -0,0 +1,38 @@
+# Lang | isTruthy
+
+## Description
+Checks if a value is of type is any other then undefined, null, false, 0, NaN or an empty string "".
```

Comment:
Maybe:
Checks if a value is truthy; Meaning: if it can be coerced into boolean `true`.
Anything not in the following list will return `true`:
- `undefined`
- `null`
- `false`
- `0`
- `NaN`
- empty string (`""`)